src/BowlingGame.js

```diff
@@ -0,0 +1,3 @@
+var BowlingGame = function () {
+
+};
```

Comment:
You could make BowlingGame a class, and then write a bowl function within that class to store properties of the game (such as an array of bowls) and then access those for scoring.